schemas/context/subscription.schema.json

```diff
@@ -160,5 +160,5 @@
       "$ref": "#/definitions/subscription"
     }
   ],
-  "meta:status": "experimental"
+  "meta:status": "stabilizing"
```

Comment:
👍

Reply:
👎 I can't how Adobe could use this with our subscriptions (eg. CC and DC), so I have to believe that this is not anywhere near stable.